packages/blockchain-link-types/src/blockbook-api.ts

```diff
@@ -1,5 +1,7 @@
 /* Do not change, this code is generated from Golang structs */
```

Comment:
I believe this file should not be changed manually as it originates from [Blockbook repo](https://github.com/trezor/blockbook/blob/master/blockbook-api.ts). Make the necessary changes in `blockbook.ts ` file instead (if they're really necessary, which may not be this case; see the main comment).